backends/qualcomm/serialization/qc_compiler_spec.fbs

```diff
@@ -34,6 +34,7 @@ table HtpInfo {
 enum QcomChipset: int {
   UNKNOWN_SM = 0,
   SA8295 = 39,
+  SM8350 = 35,
```

Comment:
@shewu-quic I look it up and it seems like adding a value in enum is not BC breacking...so the previous PR is also find to add it in the middle. What do you think?